daemon/kmd/api/cors.go

```diff
@@ -35,6 +35,10 @@ func corsMiddleware(allowedOrigins []string) func(http.Handler) http.Handler {
 			for _, allowedOrigin := range allowedOrigins {
 				if subtle.ConstantTimeCompare([]byte(origin), []byte(allowedOrigin)) == 1 {
 					found = allowedOrigin
+					break
+				} else if allowedOrigin == "*" {
+					found = origin
+					break
```

Comment:
nit: You might want to consider removing the second `break` so that explicit domains would supersede all domains.

Reply:
Why?  That changes nothing.

Reply:
If you have the following allowedOrigins list:
"*",
"r1.algorand.network",
"r2.algorand.network"

Then it would prefer the last two over the "*".

Reply:
arguably, if someone already configuring it for "*", it doesn't really matter.